codeforces/src/database/data_classes.py

```diff
@@ -85,7 +85,9 @@ def __getitem__(self, key):
         return self.__dict__[key]
 
     def dict(self) -> dict:
-        return {key: self[key] for key in self.attributes}
+        student = {key: self[key] for key in self.attributes}
+        student["city"] = self.city_name
+        return student
```

Comment:
You can use a **merge** operator here, 
E.g.:  `return {key: self[key] for key in self.attributes} | {"city": self.city_name}`